### DISCUSSION TITLE
Can't use text-white

### DISCUSSION THREAD

**Discussion by marcoagalvao** [tailwindlabs/tailwindcss#11197] [Help] [answered]
Hi guys, I have a question. I'm new to the framework and I'm not able to use the text-white property. Actually, I can't use any of the default Tailwind colors, only the ones I customized. What should I do?

**Comment by wongjn** [comment, marked as answer]
Sounds like you added your colors in `theme.colors` instead of `theme.extend.colors`. The former will completely override the default Tailwind colors, whereas the later *adds* to the default Tailwind colors.

**Reply by marcoagalvao** [reply]
Thank you very much, I didn't know it worked that way. But now, paying attention, it makes sense haha, it worked!
